python/GafferSceneUI/TransformToolUI.py

```diff
@@ -101,6 +105,22 @@ def _distance( ancestor, descendant ) :
 
 	return result
 
+class _TargetTipWidget( GafferUI.Frame ) :
+
+	def __init__( self, tool, **kw ) :
+
+		GafferUI.Frame.__init__( self, borderWidth = 4, **kw )
+
+		labels = Gaffer.Metadata.value( tool, "ui:tool:transformToolTips" )
+
+		if labels is not None :
+			with self :
+				with GafferUI.ListContainer( orientation = GafferUI.ListContainer.Orientation.Vertical, spacing = 4 ) :
+					for label in labels:
+						GafferUI.Label( label )
+		else :
```

Comment:
I haven't been able to sort out why this is not hiding the tooltip widget in the case of activating the scale tool, which doesn't do targeting. Is there a subwidget other than `self` that I should be hiding instead? I always seem to get a small box regardless of what I've tried hiding.